config.js

```diff
@@ -704,6 +704,7 @@ config.BUCKET_LOG_CONCURRENCY = 10;
 ////////////////////////////////
 config.NOTIFICATION_LOG_NS = 'notification_logging';
 config.NOTIFICATION_LOG_DIR = process.env.NOTIFICATION_LOG_DIR;
+config.NOTIFICATION_BATCH = process.env.BATCH;
```

Comment:
Why can't we do it here instead of providing default value ,10 at the time of usage ?
config.NOTIFICATION_BATCH = process.env.BATCH || 10;